src/controllers/taskInstanceController.js

```diff
@@ -0,0 +1,23 @@
+function taskInstanceController({ taskInstanceService, httpErrors }) {
+  async function getAll(req, res) {
+    const tasks = await taskInstanceService.getAll();
+    return res.json(JSON.parse(tasks));
+  }
+
+  async function updateOne(req, res, next) {
+    const updatedtask = req.body;
+    const { id } = req.params;
+    const result = await taskInstanceService.updateOne(id, updatedtask);
```

Comment:
Let's pass `req.body` as param to `updateOne` and it will save a const

```js
const result = await taskInstanceService.updateOne(id, req.body);
```